src/components/avatars/OcAvatar.vue

```diff
@@ -4,8 +4,10 @@
     :username="userName"
     :size="width"
     :src="src"
-    :alt="accessibleLabel"
-    :aria-hidden="accessibleLabel === ''"
+    :aria-label="accessibleLabel === '' ? null : accessibleLabel"
+    :aria-hidden="accessibleLabel === '' ? 'true' : 'false'"
```

Comment:
Can't we just fall back to `null` instead of `'false'` here as well, like with the other attributes?